Python/pathconfig.c

```diff
@@ -283,6 +246,54 @@ config_init_module_search_paths(PyConfig *config, _PyPathConfig *pathconfig)
 }
 
 
+/* Calculate the path configuration:
+
+   - base_executable (Windows only)
```

Comment:
Might be worth noting that it is used on all platforms, but only requires calculation at this point for Windows (which I think is what you're implying?)

Reply:
I completed the comment.